### PR TITLE
Simplify text and Elminitate Redundancy for JLIB_RULES_SETTING_NOTES

### PR DESCRIPTION
**Pull request by brianteeman**
This PR (thanks to @rcheesley) simplifies the text for this field which is seen on the permissions tab for all components.
### Before

before
### After

after
### Testing

The easiest way to check this is to go to Global configuration and then work through all the components on the left hand side and check their Permissions tab

---

Extra Reading on Eliminate redundancy and Avoiding over-communication
https://msdn.microsoft.com/en-gb/library/windows/desktop/dn742478.aspx
Eliminate redundancy
Redundant text not only takes valuable screen space, but weakens the effectiveness of the important ideas or actions that you are trying to convey. It is also a waste of the reader's time, and all the more so in a context where scanning is the norm. Windows strives to explain what users need to do once—well and concisely.
Review each window and eliminate duplicate words and statements, both within and across controls. Don't avoid important text—be explicit wherever necessary—but don't be redundant and don't explain the obvious.
Avoid over-communication
Even if text isn't redundant, it can simply be too wordy in an effort to explain every detail. Too much text discourages reading—the eye tends to skip right over it—ironically resulting in less communication rather than more. In UI text, concisely communicate the essential information. If more information is necessary for some users or some scenarios, provide a link to more detailed Help content, or perhaps to a glossary entry for clarification of a term.

### CODE REVIEW

**Comment by MAT978** [comment]
#7469 works as described (wasn't able from patchtester).
Great improvement!
Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7469">issues.joomla.org/joomla-cms/7469</a>.</sub>

**Comment by jwaisner** [comment]
@test 

PR works as intended. Always happy to reduce clutter.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7469">issues.joomla.org/joomla-cms/7469</a>.</sub>

**Comment by Bakual** [comment]
It's not exactly correct.
Especially  **Inherit** is either coming from global setting *or* from the parent group. If either allows the action, it is allowed.
For **Denied**, it ignores whatever is set in global setting *and* parent group. It's also worth to know that it will also deny the action for all children fo the group.

Other than that, it looks fine.

**Comment by brianteeman** [comment]
rebased to resolve merge conflicts

**Comment by brianteeman** [comment]
@RCheesley do you want to take another pass at the text in light of @bakual comments (when you are back from holiday)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7469">issues.joomla.org/joomla-cms/7469</a>.</sub>

**Comment by brianteeman** [comment]
Suggested change from @RCheesley taking into account @bakual comments

> 1. Changes apply to this component only.<br /><em><strong>Inherited</strong></em> - a Global Configuration setting or higher level setting is applied.<br /><em><strong>Denied</strong></em> always wins - whatever is set at the Global or higher level and applies to all child elements.<br /><em><strong>Allowed</strong></em> will enable the action for this component unless it is overruled by a Global Configuration setting.

???<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7469">issues.joomla.org/joomla-cms/7469</a>.</sub>

**Comment by Bakual** [comment]
Yep, sounds accurate this way.

**Comment by brianteeman** [comment]
OK I will update the PR accordingly

**Comment by brianteeman** [comment]
PR updated as above

**Comment by Bakual** [comment]
Looks fine, thanks!
